homeassistant/components/zha/number.py

```diff
@@ -971,6 +972,28 @@ class ThermostatLocalTempCalibration(ZHANumberConfigurationEntity):
     _attr_icon: str = ICONS[0]
 
 
+@MULTI_MATCH(
+    cluster_handler_names=CLUSTER_HANDLER_THERMOSTAT,
+    models={"TRVZB"},
+    stop_on_match_group=CLUSTER_HANDLER_THERMOSTAT,
+)
+# pylint: disable-next=hass-invalid-inheritance # needs fixing
+class ThermostatLocalTempCalibrationSonoffTRVZB(ZHANumberConfigurationEntity):
```

Comment:
I think we could also inherit `ThermostatLocalTempCalibration` instead of `ZHANumberConfigurationEntity`.

Then, we would only need to override the attributes that we actually want to change:
`_attr_native_min_value`, `_attr_native_max_value`, and `_attr_native_step`.